tests/regression/tests/REQUEST-933-APPLICATION-ATTACK-PHP/933160.yaml

```diff
@@ -421,28 +454,31 @@ tests:
             uri: /print_r
           output:
             log_contains: id "933160"
-  - test_title: 933160-27
-    desc: func\t/*foo\r\nbar*/\t (
-    stages:
-      - stage:
-          input:
-            data: x=Print_r%28%20%29
-            dest_addr: 127.0.0.1
-            headers:
-              Host: localhost
-              User-Agent: "OWASP CRS test agent"
-              Accept: text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5
-            method: POST
-            port: 80
-            uri: /astrrev()
-          output:
-            log_contains: id "933160"
+  # removing: this test is not valid, as the function doesn't exist in PHP
```

Comment:
Remove it. No sense in testing stuff that we clearly aren't looking for.

Reply:
removed